### ISSUE TITLE
Arrow keys in dropdowns

### ISSUE THREAD

**Issue by jolamar** [indiana-university/rivet-react#1]
When dropdown is opened user should be able to use up and down arrows to navigate dropdown items and it should wrap.

**Comment by jhoerr** [comment]
@jwbennet @jolamar Is it reasonable to get this done for 0.3.0 or should we punt to 0.4.0?

**Comment by jolamar** [comment]
Here are the accessibility requirements https://davatron5000.github.io/a11y-nutrition-cards/#menu-menu-button

**Comment by jwbennet** [comment]
My vote would be the opposite actually.  Would be be non-compliant with WCAG 2.0 AA if we don't support arrow keys?  If not I'd think this would be another area where we are not up to par with Rivet 1.0 yet, but would be something that can be tackled before a "1.0" release.

**Comment by jolamar** [comment]
relates to #138

**Comment by jolamar** [comment]
<img width="1220" alt="screen shot 2018-10-15 at 2 09 15 pm" src="https://user-images.githubusercontent.com/1017119/46969439-07cdfa00-d084-11e8-936c-28602a137ad9.png">
should behave as such https://davatron5000.github.io/a11y-nutrition-cards/#menu-menu-button
